kaldb/src/main/java/com/slack/kaldb/chunk/ReadWriteChunk.java

```diff
@@ -188,68 +179,49 @@ public void commit() {
     logStore.refresh();
   }
 
+  // Snapshot methods
   public void preSnapshot() {
-    LOG.info("Started RW chunk pre-snapshot {}", chunkInfo);
+    logger.info("Started RW chunk pre-snapshot {}", chunkInfo);
     setReadOnly(true);
     commit();
-    LOG.info("Finished RW chunk pre-snapshot {}", chunkInfo);
+    logger.info("Finished RW chunk pre-snapshot {}", chunkInfo);
   }
 
+  public abstract void postSnapshot();
```

Comment:
Each of the abstract methods should have a javadoc explaining their use as well in addition to the overall class doc. IntelliJ surfaces this info (if available) when you go to extend the use, which makes it easier to reference.

Reply:
Done.